manifests/load_balancer.pp

```diff
@@ -8,10 +8,25 @@
 # [*listen_addr*]
 #   The address that seed_stack::router is listening on. This prevents the more
 #   specific listen directive in that from masking our server blocks.
+#
+# [*nginx_manage*]
+#   Set to false to avoid managing the nginx package.
+#
 class seed_stack::load_balancer (
-  $listen_addr = $seed_stack::params::router_listen_addr,
+  $listen_addr  = $seed_stack::params::router_listen_addr,
+  $nginx_manage = true,
 ) inherits seed_stack::params {
   validate_ip_address($listen_addr)
+  validate_bool($nginx_manage)
+
+  if $nginx_manage {
+    package { $seed_stack::params::nginx_package:
```

Comment:
I considered that, but decided that "explicit is better than implicit". I suppose in this case we're using exactly the same parameters for both, so it's unlikely to be a problem.